tests/many_to_one/models.py

```diff
@@ -98,3 +98,18 @@ class School(models.Model):
 
 class Student(models.Model):
     school = models.ForeignKey(School)
+
+
+class UnsavedForeignKey(models.ForeignKey):
```

Comment:
Do you know of an example of this happening in another test which I can use as a reference?